libs/product/driver/shopify/src/models/shopify-product.ts

```diff
@@ -0,0 +1,13 @@
+import { ProductImages } from './product-images';
+import { ProductPriceRange } from './product-price-range';
+
+export interface ProductNode {
```

Comment:
All these models should be prefixed with `Shopify` since they specifically refer to the shape of data on the shopify backend. So this would be `ShopifyProductNode`.